Removed extra / that caused the usersList.html template to not be fou… (#643)
crate/crate-admin: CHANGES.rst

Unreleased
==========

> - Fixed an issue that caused the users list template to not be available when

Reviewer: ```suggestion
- Fixed an issue that caused the users list in the privileges tab to not displayed when
```

>   cratedb is not served from ``/``.

Reviewer: ```suggestion
  the CrateDB Admin UI is not served from ``/``.
```
